src/api/auth/url.js

```diff
@@ -0,0 +1,14 @@
+import {getBaseUrl, getInnerBaseUrl} from "../../common/environment.js";
+import {CONFIG} from "../../common/envVars.js";
+
+const baseUrl = getBaseUrl();
+const innerBaseUrl = getInnerBaseUrl();
+
+const authConfig = {
+  innerBaseUrl: `${innerBaseUrl}/p4paauth/payhub`,
+  baseUrl: CONFIG.USE_INTERNAL_ACCESS_ENV
+      ? `${innerBaseUrl}/p4paauth/payhub`
+      : `${baseUrl}/payhub-auth`
+};
```

Comment:
don't replicate authInnerBaseUrl construction
```suggestion
const authInnerBaseUrl = `${innerBaseUrl}/p4paauth/payhub`;
const authConfig = {
  innerBaseUrl: authInnerBaseUrl,
  baseUrl: CONFIG.USE_INTERNAL_ACCESS_ENV
      ? authInnerBaseUrl
      : `${baseUrl}/payhub-auth`
};
```